newton/_src/sim/builder.py

```
@@ -11407,6 +11432,13 @@ def _to_wp_array(data, dtype, requires_grad):
             m.constraint_mimic_joint1 = wp.array(self.constraint_mimic_joint1, dtype=wp.int32)
             m.constraint_mimic_coef0 = wp.array(self.constraint_mimic_coef0, dtype=wp.float32)
             m.constraint_mimic_coef1 = wp.array(self.constraint_mimic_coef1, dtype=wp.float32)
+            m.constraint_mimic_stiffness = wp.array(self.constraint_mimic_stiffness, dtype=wp.float32)
+            m.constraint_mimic_damping = wp.array(self.constraint_mimic_damping, dtype=wp.float32)
+            # Host-side flag so solvers can warn without a device copy at init.
+            m._has_authored_mimic_compliance = any(
+                math.isfinite(s) and s > 0.0 and math.isfinite(d) and d > 0.0
+                for s, d in zip(self.constraint_mimic_stiffness, self.constraint_mimic_damping)
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)